codestodians/komlie-frontend
# Add  and Implement forgotPassword page

#### What does this PR do?
Shows the Implementation of Forgot-password Page
Fix issue with delayed rendering of Components

#### Description of Task to be completed?
 A user should be able to change password if forgotten

#### How should this be manually tested?
- Sign up with a valid Email
- Click on Login Page to reset
- Enter Email.
- The mail should be available in your inbox

#### Any background context you want to provide?
Note that the content of the mail reflects `mealdock`. This will be changed in another PR

#### What are the relevant pivotal tracker stories?
[#166791294](https://www.pivotaltracker.com/story/show/166791294)
#### Screenshots (if appropriate)

#### Questions:
